code/modules/admin/admin_attack_log.dm

```diff
@@ -1,5 +1,5 @@
-/mob/var/lastattacker = null
-/mob/var/lastattacked = null
+/mob/var/mob/lastattacker = null
+/mob/var/mob/lastattacked = null
```

Comment:
is this handled in destroy at all?

Reply:
I didn't check, as this is somewhat out of the scope for that pr.